proto/eth/v2/beacon_block.proto

```diff
@@ -72,10 +75,11 @@ message SignedBlindedBeaconBlockContainer {
     v1.BeaconBlock phase0_block = 1;
     BeaconBlockAltair altair_block = 2;
     BlindedBeaconBlockBellatrix bellatrix_block = 3;
+    BlindedBeaconBlockEip4844 eip4844_block = 4;
   }
 
   // 96 byte BLS signature from the validator that produced this block.
-  bytes signature = 4 [(ethereum.eth.ext.ssz_size) = "96"];
+  bytes signature = 100 [(ethereum.eth.ext.ssz_size) = "96"];
```

Comment:
Shouldn't the field number here be `5` not `100`?

Reply:
I think this is safe even for backwards compatibility because these messages are presented via the Beacon API as JSONs not protobufs.